drivers/SmartThings/zigbee-contact/src/init.lua

```diff
@@ -45,7 +45,6 @@ local zigbee_contact_driver_template = {
   },
   sub_drivers = {
     require("aqara"),
-    require("battery-overrides"),
```

Comment:
Is this now released in 51?

Reply:
It wasn't for 51. It was just a sub-driver for one specific Visonic device. Behavior could have been captured by simply adding the model to drivers/SmartThings/zigbee-contact/src/contact-temperature-sensor/init.lua, so that's what I did.